apps/ui/src/Pages/RidesFeed/CreateRideDialog.tsx

```diff
@@ -65,6 +65,23 @@ const CreateRideDialog = ({
     setOpen(false);
   };
 
+  const handleGasSeatsChange = (fieldName: string, value: string) => {
+    switch (fieldName) {
+      case 'gasMoney':
+        setGasMoney(value);
+        break;
+      case 'seats':
+        setSeats(value);
+        break;
+      default:
+        break;
+    }
+  };
+  const isGasMoneyNegative = Number(gasMoney) < 0;
+  const isSeatsLessThanOne = Number(seats) < 1;
+  const isSeatsNotInteger = !Number.isInteger(Number(seats));
+
+
```

Comment:
I recommend seeing how validations were done during the registration process or connecting to the system

Reply:
In addition, consider saving the validation logic in a separate validation file (there is one somewhere) and only call the function with the parameters.
It is better for the component not to know what validation is being done, but only to perform validation

Reply:
added external validation file for rides